input/docs/controls/ColorPicker.md

```diff
@@ -0,0 +1,492 @@
+| `MahApps.Styles.Slider.ColorComponent.SV`  | `Style`        | overrides the `Style` of the S- and V-`Slider` |
+| `MahApps.Styles.ColorEyeDropper.ColorCanvas` | `Style`        | overrides the Style of the `ColorEyeDropper` |
+
+
+## Example
+
+```xaml
+<!-- make sure to add the right namespace -->
+<!-- xmlns:mah="http://metro.mahapps.com/winfx/xaml/controls" -->
+
+<mah:ColorCanvas x:Name="ColorCanvasExample"
+                 SelectedColor="Blue"
+                 DefaultColor="Transparent"
+                 LabelAlphaChannel="Alpha"
+                 LabelBlueChannel="Blue"
+                 LabelGreenChannel="Green"
+                 LabelRedChannel="Red" />
+```
+
+# ColorPalette
+The `ColorPalette` can be used to present the user a swatch of predefined colors. As this control is derived from `System.Windows.Controls.ListBox` you can use all functionality know from the `ListBox`-Control
+
+## The user interface
+
+![](images/ColorPicker_ColorPalette_Numbered.png)
+
+| No | Description                              |
+|----|------------------------------------------|
+| 01 | The header of the `ColorPalette`         |
+| 02 | Displays the available colors            |
+| 03 | The currently selected color is highlighted |
+
+## Properties
+The available colors can be either added directly to the `Items`-property or by binding to the `ItemsSource`-property. The selection can be handled by binding to `SelectedValue`, `SelectedItem` or `SelectedIndex`. 
+
+In addition to this the `ColorPalette` provides the following properties.
+
+| Property             | Type                       | Description                              |
+|----------------------|----------------------------|------------------------------------------|
+| Header               | `object`                     | Gets or sets the header content          |
+| HeaderTemplate       | `DataTemplate`               | Gets or sets the header template         |
+| ColorNamesDictionary | `Dictionary<Color?, string>` | Gets or sets the `Dictionary<Color?, string>` used to get or set the ColorName [(see also)](#colornamesdictionary-and-colorhelper) |
+
+## DynamicResources
+| Key                                      | Type         | Description                              |
+|------------------------------------------|--------------|------------------------------------------|
+| `MahApps.Sizes.ColorListBox.ItemWidth`     | `double`       | overrides the width of the items         |
+| `MahApps.Sizes.ColorListBox.ItemHeight`    | `double`       | overrides the height of the items        |
+| `MahApps.Brushes.Tile.Small`               | `Brush`        | overrides the checkered Brush which is visible if the color is transparent |
+| `MahApps.Styles.ListBoxItem.ColorPaletteItem` | `Style`        | overrides the `Style` of the items       |
+| `MahApps.Templates.ColorPaletteItem`       | `DataTemplate` | overrides the `DataTemplate` of the items |
```

Comment:
Add missing style MahApps.Styles.ColorPalette.ColorPickerDropDown and MahApps.Styles.ColorPalette